$OpenBSD: patch-configure_in,v 1.4 2011/07/06 14:14:06 kili Exp $

# Builds fine with gcc 3.3.5 without makeinfo (unless you try to
# build the documentation).

--- configure.in.orig	Sun Jun 12 16:05:50 2011
+++ configure.in	Mon Jul  4 15:29:57 2011
@@ -91,10 +91,10 @@ AC_MSG_RESULT($NCSB_SOURCE_FILES)
 AC_LANG([C++])
 
 STEPMAKE_PYTHON(REQUIRED, 2.4)
-STEPMAKE_GCC(REQUIRED, 3.4)
+STEPMAKE_GCC(REQUIRED, 3.3)
 
 STEPMAKE_CXX(REQUIRED)
-STEPMAKE_GXX(REQUIRED, 3.4)
+STEPMAKE_GXX(REQUIRED, 3.3)
 STEPMAKE_CXXTEMPLATE
 STEPMAKE_STL_DATA_METHOD
 STEPMAKE_LIB(REQUIRED)
@@ -185,7 +185,7 @@ STEPMAKE_GUILE(OPTIONAL)
 # perl for help2man.
 STEPMAKE_PERL(OPTIONAL)
 
-STEPMAKE_PROGS(MAKEINFO, makeinfo, REQUIRED, 4.11)
+STEPMAKE_PROGS(MAKEINFO, makeinfo, OPTIONAL, 4.11)
 STEPMAKE_PROGS(TEXI2HTML, texi2html, OPTIONAL, 1.82)
 STEPMAKE_PROGS(DBLATEX, dblatex, OPTIONAL)
 STEPMAKE_PROGS(PDFLATEX, pdflatex, OPTIONAL)
